cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/RecordWriter.java

```diff
@@ -108,12 +109,23 @@ private String getStreamId(TaskContext taskContext)
     public List<StreamResult> write(Iterator<Tuple2<DecoratedKey, Object[]>> sourceIterator)
     {
         TaskContext taskContext = taskContextSupplier.get();
+        LOGGER.info("[{}]: Processing bulk writer partition", taskContext.partitionId());
+
         Range<BigInteger> taskTokenRange = getTokenRange(taskContext);
         Preconditions.checkState(!taskTokenRange.isEmpty(),
                                  "Token range for the partition %s is empty",
                                  taskTokenRange);
 
         TokenRangeMapping<RingInstance> initialTokenRangeMapping = writerContext.cluster().getTokenRangeMapping(false);
+        LOGGER.info("[{}]: Fetched token range mapping for keyspace:{} with write replicas:{} containing pending " +
+                    "replicas:{}, blocked instances: {}, replacement instances: {}",
```

Comment:
```suggestion
        LOGGER.info("[{}]: Fetched token range mapping for keyspace: {} with write replicas: {} containing pending " +
                    "replicas: {}, blocked instances: {}, replacement instances: {}",
```

Reply:
debug level?

Reply:
This is intended to be info level so we can reason about the bulk write decisions in the job. The log has summarized counts for this purpose.

We do have a similar more verbose debug log where the request is made.